microshift_install_get_ready/microshift-install-get-ready.adoc

```diff
@@ -16,3 +16,70 @@ include::modules/microshift-install-system-requirements.adoc[leveloffset=+1]
+** See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_logical_volumes/overview-of-logical-volume-management_configuring-and-managing-logical-volumes[Overview of logical volume management] for more information about creating volume groups and persistent volumes on {op-system-base}.
+** See xref:../microshift_storage/microshift-storage-plugin-overview.adoc#microshift-storage-plugin-overview[Dynamic storage using the LVMS plugin] for more information about the {microshift-short} plug-in.
+
+* Configure networking settings according to the access needs you plan for your {microshift-short} cluster and applications. Consider whether you want to use single or dual-stack networks, configure a firewall, or configure routes.
+** See xref:../microshift_networking/microshift-networking-settings.adoc#microshift-networking[Understanding networking settings] for more information about {microshift-short} networking options.
+//Q: should we have a RHEL link here as well?
+
+* Install the {oc-first} to access your cluster, see xref:../microshift_cli_ref/microshift-oc-cli-install.adoc#microshift-oc-cli-install[Installing the OpenShift CLI tool].
+
+* Understand the differences between the valid {op-system-base} installation types available, along with the support scope for each and the tools used.
+
+** Often, {op-system-ostree} is chosen for end-point devices with production clusters. This installation type can be done with network connections or offline, depending on the local environment.
+** Image mode for {op-system-base} is available as Technical Preview.
+** Sometimes, {op-system-rt-kernel} can be used where predictable latency is critical. Workload partitioning is also required for low-latency applications. See xref:../microshift_configuring/microshift_low_latency/microshift-low-latency.adoc#microshift-low-latency[Configuring low latency] for more information on low latency and the {op-system-rtk}.
+
+* When choosing an image-based installation, consider whether the destination hardware is intended to be in an offline or networked state, where you plan to build system images, and how you plan to load your {op-system-bundle} onto the hardware. Use the following scenarios as general guidance:
+
+** If you build either a fully self-contained {op-system-ostree} or image mode for {op-system-base} ISO outside a disconnected environment, then install the ISO locally on your edge devices, you likely do not need an RPM repository or a mirror registry.
+** If you build an ISO outside a disconnected environment that does not include the container images, but consists of only the RPMs, you need a mirror registry from which to pull container images.
+** If you build images inside a disconnected environment, or use package mode for installations, you need both a mirror registry and a simple file or directory-based RPM repository.
+
+Using RPMs, or package-based installation, of {microshift-short}::
```

Comment:
This shows as a heading in my preview, is that intended or a bug in the preview? I confuses me a bit. 
This is the installation type "package mode", I think we should pull this up and introduce earlier (see above comments)